Alpaca.Markets/PolygonDataClient.cs

```diff
@@ -250,7 +250,38 @@ public sealed class PolygonDataClient : IDisposable
                 <IHistoricalItems<IAgg>, JsonHistoricalItems<IAgg, JsonMinuteAgg>>(
                     FakeThrottler.Instance, builder, cancellationToken);
         }
+         /// <summary>
+        /// Gets list of historical hour bars for single asset from Polygon's v2 REST API endpoint.
+        /// </summary>
+        /// <param name="symbol">>Asset name for data retrieval.</param>
+        /// <param name="multiplier">>Number of bars to combine in each result.</param>
+        /// <param name="dateFromInclusive">Start time for filtering (inclusive).</param>
+        /// <param name="dateToInclusive">End time for filtering (inclusive).</param>
+        /// <param name="unadjusted">Set to true if the results should not be adjusted for splits.</param>
+        /// <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
+        /// <returns>Read-only list of minute bars for specified asset.</returns>
+        public Task<IHistoricalItems<IAgg>> ListHourlyAggregatesAsync(
```

Comment:
I know that `Hourly` looks much better as a part of the method name but other methods use `Day` and `Minute` for specifying aggregation period so I think it would be better to use the same approach here too.